docs/changes/table/15956.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix an incorrect error message in ``io.votable.TableElement.parse``.
```

Comment:
We don't usually record error message updates in the change log, but in this case the old error message was wrong, so an entry might not be a bad idea after all. However, if there is an entry then it should be detailed enough that a user reading it would understand under what circumstances the erroneous message appeared what was wrong about it.